source/features/filter-pr-by-build-status.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {checkInline} from '../libs/icons';
+import features from '../libs/features';
+import {getOwnerAndRepo} from '../libs/utils';
+
+function init() {
+	const filterAssignee = select('.table-list-filters > .table-list-header-toggle:last-child > details:nth-child(7)');
+
+	filterAssignee.parentNode.insertBefore(
+		<details class="details-reset details-overlay float-left select-menu">
+			<summary class="btn-link select-menu-button" aria-haspopup="menu">
+				Status&nbsp;
+			</summary>
+			<details-menu class="select-menu-modal position-absolute right-0" style={{zIndex: 99}} role="menu">
+				<div class="select-menu-header">
+					<span class="select-menu-title">Filter by Status</span>
+				</div>
+				<div class="select-menu-list">
+					{getStatusDropDown()}
+				</div>
+			</details-menu>
+		</details>,
+		filterAssignee
+	);
+}
+
+const getStatusDropDown = () => {
+	const {ownerName, repoName} = getOwnerAndRepo();
+	const baseUrl = `https://github.com/${ownerName}/${repoName}/pulls`;
```

Comment:
This line isn't needed (as described in another comment), but keep in mind that RGH also works on GitHub Enterprise. It's best to use relative URLs (starting with `/` or `?` where possible)

Reply:
Good to know. I wasn't aware of enterprise GitHub for RGH.